### DISCUSSION TITLE
Error in TPCDS tests.

### DISCUSSION THREAD

**Discussion by sumokiama** [facebookincubator/velox#6080] [Q&A]
It appears that at the end of a number of TPCDS tests, Velox fails. Not sure if this matters as we do get a result for each test.

E0809 23:12:41.068053 124663 Exceptions.h:68] Line: ../../velox/exec/Task.cpp:1476, Function:terminate, Expression:  Cancelled, Source: RUNTIME, ErrorCode: INVALID_STATE

**Comment by majetideepak** [comment]
@sumokiama How are you running the TPCDS tests? Do you have a reproducer you can share?

**Reply by sumokiama** [reply]
Here are the scala and shell scripts I am using. Since the repo does not include a tpcds shell script, I modified the one for tpch.
[tpcds-scripts.zip](https://github.com/facebookincubator/velox/files/12337978/tpcds-scripts.zip)
